kobo/apps/hook/views/v2/hook_log.py

```diff
@@ -38,6 +38,12 @@ class HookLogViewSet(AssetNestedObjectViewsetMixin,
     * `hook_uid` - is the unique identifier of a specific external service
     * `uid` - is the unique identifier of a specific log
 
+    Use `status` query parameter to filter logs by numeric status:
+
+    * `status=0`: hook has failed after exhausting all retries
+    * `status=1`: hook is still pending
+    * `status=2`: hook has succeeded
```

Comment:
Do you mean in the API, or in the code?
The code does not currently use the values, but I agree it should, for validation.

However, in the API, the problem is that the numeric ids are what's currently returned by the API, therefore I believe this should also be what the parameters take.
I think it would be strange to send a request with `status=failed` and then receive a response with a bunch of objects having a `{status: 0}` field...

Reply:
🤔  you are right.
This should be changed as well. (and update front-end codes too)